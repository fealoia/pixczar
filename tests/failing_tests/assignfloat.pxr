Void main() {
	Float f  = 2.0;
	f = 3;
}
