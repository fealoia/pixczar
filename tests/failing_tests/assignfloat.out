Fatal error: exception Failure("illegal assignment Float = Int in f = 3")
